lib/index.js

```diff
@@ -1,21 +1,65 @@
-export default (callback) => (input) => {
+export default (opts = {}, callback) => (input) => {
```

Comment:
what do you think about this:

```js
export default (filesGlob, opts = {}) => (callback) => () => {
```

this way there is no need to guess how many arguments we have.

also it'd be a breaking change anyway so let remove supporting of `input` as well – there should be only one way to pass files glob with no confusion.

```js
watch('lib/**/*.js', /* options */)((file) => start(
  files(file),
  read(),
  babel(),
  write('build/')
))
```

also, we have a kinda abstract helper [input-connector](https://github.com/start-runner/input-connector) which should be used instead of that `files(file)` "hack" – but we need to make sure that array and not a single value is passed.